type-generator/package.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Can we consider having just a single set of dependencies for all the TS/JS in this this repo? It feels like having to maintain this might be a bit burdensome and we might be able to just have a single package.json in the top level?

Reply:
Hi @galund, I've reworked this now to have a single JS module in the repo instead of the sub-project.